### ISSUE TITLE
DnnSuperResImpl throws java.lang.UnsatisfiedLinkError

### ISSUE THREAD

**Issue by cansik** [bytedeco/javacv#1396]
When I try to use `DnnSuperResImpl` with the current snapshot (1.5.3-SNAPSHOT) I immediately get the following error:

```java
java.lang.UnsatisfiedLinkError: no jniopencv_dnn_superres in java.library.path
```

Here is an example code:

```java
DnnSuperResImpl net = new DnnSuperResImpl();
net.readModel(model.toAbsolutePath().toString());
net.setModel("fsrcnn", 2);
```

I have even tried to load the library with:

```java
Loader.load(opencv_java.class);
Loader.load(opencv_dnn_superres.class);
```

Is this library not bundled in the opencv preset build or am I using it the wrong way?

**Comment by saudet** [comment]
Works fine here on Linux. What's your platform?

**Comment by saudet** [comment]
It looks like there is a missing inheritance on `opencv_quality` here though:
https://github.com/bytedeco/javacpp-presets/blob/master/opencv/src/main/java/org/bytedeco/opencv/presets/opencv_dnn_superres.java#L36
Please let me know if it helps to call `Loader.load(opencv_quality.class)` before. Thanks!

**Comment by cansik** [comment]
Perfect, using `Loader.load(opencv_quality.class)` fixed it. I am working on MacOS (Catalina 10.15.3).

**Comment by saudet** [comment]
Fix included in just released version 1.5.3. Thanks for reporting and for testing!
